src/support.cpp

```diff
@@ -924,8 +924,9 @@ void AreaSupport::generateSupportAreasForMesh(SliceDataStorage& storage,
                                                    // specs also have small circumference) and we want a small area to differentiate between larger blobs and "specs" in
                                                    // polygon (both have a circumference but only "specs" have a "larger blobs" both have a small circumference but only
                                                    // "specs" has a small area).
-                                                   constexpr size_t min_circumference = 500;
-                                                   constexpr double min_area = 0.1;
+                                                   const auto nozzle_diameter = mesh_group_settings.get<coord_t>("machine_nozzle_size");
+                                                   const coord_t min_circumference = nozzle_diameter * M_PI;
+                                                   const double min_area = INT2MM2(10 * (nozzle_diameter * nozzle_diameter) / (4 * M_PI));
```

Comment:
I think this should be (so mulitply with pi instead of dividing by pi):
INT2MM2((nozzle_diameter * nozzle_diameter) / 4 * M_PI);

Reply:
https://github.com/Ultimaker/CuraEngine/pull/1803/commits/7340a0fcd8eea310c7380640ac64dad06289415c